hack/images/ci/Dockerfile

```diff
@@ -23,6 +23,9 @@ ARG GOARCH
 ARG GOPROXY
 ENV GOOS=${GOOS:-linux} GOARCH=${GOARCH:-amd64}
 ENV GOPROXY ${GOPROXY:-https://proxy.golang.org}
+# Install kubebuilder (required for unit tests)
+RUN curl -sL https://go.kubebuilder.io/dl/2.3.1/linux/amd64 | tar -xz -C /tmp/ \
```

Comment:
we won't be able to remove this for now, otherwise the unit test will fail
I can make this a follow-up item with refactoring unit tests to not use ginkgo/envtest